minard/views.py

```diff
@@ -551,10 +551,19 @@ def l2():
 @app.route('/burst')
 def burst():
     offset = request.args.get('offset',type=int)
-    limit = request.args.get('limit',default=15,type=int)
+    limit = request.args.get('limit',default=25,type=int)
+    search = request.args.get('search',type=str)
+    if search is not None:
+        start = request.args.get('start')
+        end = request.args.get('end')
+        if offset == None:
+            return redirect("burst?limit="+str(limit)+"&offset=0&search="+search+"&start="+str(start)+"&end="+str(end))
```

Comment:
Should use string formatting here, i.e. `burst?limit=%i&offset=" % (limit,...)`

Reply:
done